_rules/descriptive-label-cc0f0a.md

```diff
@@ -158,30 +158,43 @@ Label is included in accessibility tree, but not visible, and does not describe
 `Label` that is neither visible to users, nor included in the accessibility tree.
 
 ```html
-<label for="fname" style:"display:none;">First name:</label>
-<input id="fname" type="text" name="fname"/>
+<div style="display:none">
+  <label for="bad_label">Menu:</label>
+  <input id="fname" type="text" name="bad_label"/>
+</div>
```

Comment:
Wrapped the input in a hidden element too, and made this be a "fail" if the element was visible, to distinguish it from pass.